sample/cal/src/settings.ts

```diff
@@ -1,5 +1,9 @@
-export const diffServerURL = 'http://localhost:7001/pull';
+export const diffServerURL = 'https://serve.replicache.dev/pull';
 export const diffServerAuth = 'sandbox';
-export const batchURL = 'http://localhost:5000/replicache-batch';
+
+const dataLayerBase = 'https://replicache-sample-gcal.herokuapp.com/';
```

Comment:
Looks like we are not setting the CORS headers correctly here:

`
Access to fetch at 'https://replicache-sample-gcal.herokuapp.com/replicache-batch' from origin 'http://localhost:5000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
`